### PR TITLE
ripe-atlas-tools: init at 2.3.0

### PR DESCRIPTION
**Pull request by RaitoBezarius**
Depends on #155267 and #155270

###### Motivation for this change

RIPE Atlas tools are command line tooling for the RIPE Atlas measurements project: https://atlas.ripe.net/measurements/ which is very useful to measure state of services across multiple point of presence.

I am unable to properly run the unit tests as it is complaining about ripe.atlas.sagan/cousteau failing imports but I am able to use the binaries nevertheless, if anyone has an idea, I'd be glad to implement it.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
Hm, we might want to honor https://github.com/RIPE-NCC/ripe-atlas-tools/blob/master/docs/packaging.rst advice which involves writing the nixpkgs version in the user-agent or the NixOS version? Unsure which one of those are relevant.

File: pkgs/tools/networking/ripe-atlas-tools/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  src = fetchFromGitHub {
+    owner = "RIPE-NCC";
+    repo = "ripe-atlas-tools";
+    rev = "v${version}";
+    sha256 = "sha256-5AMqBXxJZOtI0/2NrEjrUfNXWKc7sn6kZX26766LBUM=";
+  };
+
+  checkInputs = with python3.pkgs; [
+    sphinx
+    flake8
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/ripe-atlas-tools/default.nix:19]:
```suggestion
```

File: pkgs/tools/networking/ripe-atlas-tools/default.nix [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    pytestCheckHook
+  ];
+
+  # TODO(raitobezarius): cannot import during tests ripe.atlas.cousteau/ripe.atlas.sagan, I do not know why.
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/ripe-atlas-tools/default.nix:23]: Is the direcotry in source/PWD shadowing the importing? Try rm'ing it.

File: pkgs/tools/networking/ripe-atlas-tools/default.nix [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  propagatedBuildInputs = with python3.pkgs; [
+    ripe-atlas-cousteau
+    ripe-atlas-sagan
+
+    ujson
+
+    sphinx
+    sphinx_rtd_theme
+
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/ripe-atlas-tools/default.nix:34]:
```suggestion
    ujson
    sphinx
    sphinx_rtd_theme
```

File: pkgs/tools/networking/ripe-atlas-tools/update.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix-update
+set -eu -o pipefail
+
+nix-update ripe-atlas-tools
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/ripe-atlas-tools/update.sh:5]:
```suggestion
```
we don't need this

**Comment by RaitoBezarius** [comment]
Woops. :(

**Comment by dasJ** [comment]
Please create a new PR where there are no additional people pinged

**Pull request closed without merging**